.github/workflows/lychee.yml

```diff
@@ -1,36 +1,55 @@
     link-check:
         runs-on: ubuntu-latest
         steps:
-            -   uses: actions/checkout@v4
+            - uses: actions/checkout@v4
 
-            -   name: Use Node.js 20
-                uses: actions/setup-node@v4
-                with:
-                    node-version: 20
-                    cache: 'npm'
-                    cache-dependency-path: 'package-lock.json'
-                    always-auth: 'true'
-                    registry-url: 'https://npm.pkg.github.com/'
-                    scope: '@apify-packages'
+            - name: Use Node.js 20
+              uses: actions/setup-node@v4
+              with:
+                  node-version: 20
+                  cache: 'npm'
+                  cache-dependency-path: 'package-lock.json'
+                  always-auth: 'true'
+                  registry-url: 'https://npm.pkg.github.com/'
+                  scope: '@apify-packages'
 
-            -   name: Build docs
-                run: |
-                    npm ci --force
-                    npm update @apify/openapi
-                    npm run build
-                env:
-                    APIFY_SIGNING_TOKEN: ${{ secrets.APIFY_SIGNING_TOKEN }}
-                    NODE_AUTH_TOKEN: ${{ secrets.NPM_TOKEN }}
-                    SMARTLOOK_PROJECT_KEY: ${{ secrets.SMARTLOOK_PROJECT_KEY }}
+            - name: Build docs
+              run: |
+                  npm ci --force
+                  npm update @apify/openapi
+                  npm run build
+              env:
+                  APIFY_SIGNING_TOKEN: ${{ secrets.APIFY_SIGNING_TOKEN }}
+                  NODE_AUTH_TOKEN: ${{ secrets.NPM_TOKEN }}
+                  SMARTLOOK_PROJECT_KEY: ${{ secrets.SMARTLOOK_PROJECT_KEY }}
 
-            -   uses: lycheeverse/lychee-action@v1.10.0
-                env:
-                    GITHUB_TOKEN: ${{ secrets.APIFY_SERVICE_ACCOUNT_GITHUB_TOKEN }}
-                with:
-                    fail: true
-                    args: --base https://docs.apify.com --exclude-path 'build/versions.html' --max-retries 6 --verbose --no-progress --accept '100..=103,200..=299,403..=403, 429' './build/**/*.html'
+            - name: Run Lychee Link Checker
+              id: lychee
+              uses: lycheeverse/lychee-action@v1.10.0
+              env:
+                  GITHUB_TOKEN: ${{ secrets.APIFY_SERVICE_ACCOUNT_GITHUB_TOKEN }}
+              with:
+                  fail: false
+                  args: >
+                      --base https://docs.apify.com
+                      --exclude-path 'build/versions.html'
+                      --max-retries 6
+                      --verbose
+                      --no-progress
+                      --accept '100..=103,200..=299,403..=403,429'
+                      --output ./lychee/out.md
+                      --format markdown
+                      './build/**/*.html'
+              continue-on-error: true
+
+            - name: Add job summary
+              if: always()
```

Comment:
whats the point of this?

Reply:
My understanding is that this creates job summary that can be later found within Actions UI based on the .md file created during the run.

Reply:
i mean what is the point of using `if: always()`? removing that line should have the same effect unless i am missing something :]

Reply:
Ah fair enough, this was supposed to make sure job summary gets created even if previous steps fail, but not sure if this is correct way to do it and if you think it should be removed , let's remove it

Reply:
oh i see, never saw it before. i dont mind keeping it there if it actually does something, up to you really, maybe keep it and remove once you see its not adding value for the broke builds?

Reply:
that makes sense to me